packages/config/config/devices/0x0086/templates/aeotec_template.json

```
@@ -1127,5 +1323,529 @@
 		"valueSize": 3,
 		"unit": "%",
 		"defaultValue": 50
+	},
+	"wake_up_timeout": {
+		"label": "Wake Up Timeout",
+		"unit": "seconds",
+		"valueSize": 1,
+		"minValue": 8,
+		"maxValue": 127,
+		"defaultValue": 15,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"current_power_mode": {
+		"$if": "firmwareVersion >= 1.7 && firmwareVersion <= 1.9",
+		"label": "Current Power Mode",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"readOnly": true,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "USB power mode",
+				"value": 0
+			},
+			{
+				"label": "Battery power mode",
+				"value": 1
+			}
+		]
+	},
+	"sleep_state": {
+		"label": "Sleep State",
+		"valueSize": 2,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"readOnly": true,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Asleep",
+				"value": 0
+			},
+			{
+				"label": "Awake",
+				"value": 1
+			}
+		]
+	},
+	"switch_disconnet_s1": {
```

```suggestion
	"switch_disconnect_s1": {
```
but that doesn't seem to match the label? Same for the next one.